solr/modules/otel/README.md

```diff
@@ -0,0 +1,33 @@
+Apache Solr Open Telemetry Tracer
+=====================================
+
+Introduction
+------------
+This module brings support for the new [OTEL](https://opentelemetry.io) standard,
+and exposes a tracer configurator that can be enabled in the
+`<tracerConfig>` tag of `solr.xml`:
+
+```xml
+<tracerConfig name="tracerConfig" class="org.apache.solr.otel.OtelTracerConfigurator" />
+```
+
+The tracer can be configured with environment variables, see https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/ and users can change both the exprter, trace propagator and many other settings.
+
+The defaults are: 
+
+```
+OTEL_SDK_DISABLED=false
```

Comment:
I think I'd prefer `OPENTELEMETRY_...` but it is a bit longer. It does make it 100% clear what this is for.

Reply:
These are OTEL standardized env variables, which works across all language implementations for the clients. So we cannot change them :) See the link above.

Anyway, need to keep this README quite short, as only devs will see it. Will probably move stuff to RefGuide...